### PR TITLE
added missing dependency, npm test command, edited README.md

### PR DESCRIPTION
**Pull request by mlisbit**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ node_modules/
 .DS_Store
 assets/css-artifacts
 assets/js/bundle.js
+npm-debug.log
```
Comment (hegrec) [inline .gitignore]: Can you make sure there is a new line at the end of the file?

File: package.json [hunk 2]
```diff
@@ -40,8 +40,10 @@
     "lodash": "^3.5.0",
     "nod-validate": "^2.0.12",
     "pace": "hubspot/pace",
+    "phantomjs-prebuilt": "^2.1.7",
```
Comment (hegrec) [inline package.json]: Nice addition for NPM3 support.

**Comment by hegrec** [comment]
This looks good to me once the new line issue is fixed. We should add the other peer deps to package.json as well.

**Comment by mickr** [comment]
Can you rebase this?

**Pull request closed without merging**